src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
+                .flatMap(string -> Arrays.stream(string.split(",")))
+                .map(Integer::parseInt)
```

Comment:
reduce map operation count to one 
```suggestion
                .flatMap(string -> Arrays.stream(string.split(",")))
                .map(Integer::parseInt)
```